python/python/glide/async_commands/cluster_commands.py

```diff
@@ -478,3 +483,34 @@ async def publish(self, message: str, channel: str, sharded: bool = False) -> in
             RequestType.SPublish if sharded else RequestType.Publish, [channel, message]
         )
         return cast(int, result)
+
+    async def flushall(
+        self, flush_mode: Optional[FlushMode] = None, route: Optional[Route] = None
+    ) -> TClusterResponse[TOK]:
+        """
+        Deletes all the keys of all the existing databases. This command never fails.
```

Comment:
Add notice that it is routed to all primaries if no route given

Reply:
Will add in the description of `route` under `Args`